sympy/physics/units/tests/test_quantities.py

```diff
@@ -414,9 +414,6 @@ def test_binary_information():
 
 
 def test_conversion_with_2_nonstandard_dimensions():
-    smartness = Dimension("smartness")
-    generousness = Dimension("generousness")
```

Comment:
Seems like something was meant to be here that used these but was omitted.

Reply:
That's probably the case for many of the lines changed here. Hopefully we can pick up on these automatically with flake8 in future so that the original author can decide whether they wanted to add something else or just delete these lines.